src/common/infra/cluster/mod.rs

```diff
@@ -182,8 +182,12 @@ pub async fn register_and_keep_alive() -> Result<()> {
 
     // check node heatbeat
     tokio::task::spawn(async move {
+        let cfg = get_config();
+        let client = reqwest::ClientBuilder::new()
+            .danger_accept_invalid_certs(true)
```

Comment:
Why is this hardcoded to `True`? This creates unsecure deployments.

Reply:
this is for internal health check, no need to check the TLS. if the tls configure wrong, it will cause the whole cluster offline.